Framework/Geometry/src/Rendering/GeometryHandler.cpp

```diff
@@ -79,17 +79,18 @@ size_t GeometryHandler::numberOfPoints() {
   return 0;
 }
 
-boost::optional<const std::vector<double> &>
-GeometryHandler::getTriangleVertices() {
+const std::vector<double> &GeometryHandler::getTriangleVertices() {
+  static std::vector<double> empty;
```

Comment:
`const`?

Nice use of local `static` variables for thread-safe defaults here and above btw. 👍